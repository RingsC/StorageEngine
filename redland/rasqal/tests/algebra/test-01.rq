# http://www.w3.org/TR/2008/REC-rdf-sparql-query-20080115/#convertGraphPattern
#
# Example: group with a basic graph pattern consisting of a single triple pattern:
#
PREFIX : <http://example.org#>
SELECT * WHERE
{ ?s ?p ?o }
