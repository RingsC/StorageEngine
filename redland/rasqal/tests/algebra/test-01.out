Project(
        BGP(
            triple(variable(s), variable(p), variable(o))
        ) ,
        Variables([ variable(s), variable(p), variable(o) ])
)
